block/local/adapter.go

```diff
@@ -341,3 +351,8 @@ func (l *Adapter) GenerateInventory(_ context.Context, _ logging.Logger, _ strin
 func (l *Adapter) BlockstoreType() string {
 	return BlockstoreType
 }
+
+func isValidUploadID(uploadID string) bool {
+	_, err := strconv.ParseUint(uploadID, 16, 64)
+	return err != nil
+}
```

Comment:
Consider returning `error`, and then using this error message to wrap `ErrInvalidUploadIDFormat`.  That might give the caller more information about what went wrong.